go/vt/vttablet/tabletserver/livequeryz_test.go

```diff
@@ -30,8 +30,8 @@ func TestLiveQueryzHandlerJSON(t *testing.T) {
 	req, _ := http.NewRequest("GET", "/livequeryz/?format=json", nil)
 
 	queryList := NewQueryList("test", sqlparser.NewTestParser())
-	queryList.Add(NewQueryDetail(context.Background(), &testConn{id: 1}))
-	queryList.Add(NewQueryDetail(context.Background(), &testConn{id: 2}))
+	_ = queryList.Add(NewQueryDetail(context.Background(), &testConn{id: 1}))
+	_ = queryList.Add(NewQueryDetail(context.Background(), &testConn{id: 2}))
```

Comment:
Do we want to instead `require.NoError` for the returns here? Would that help any future potential surprises if we also could return an error for other reasons?

Reply:
Fixed